demo/shapes/src/foreign-object.js

```diff
@@ -57,16 +60,21 @@ const Example = joint.dia.Element.define('example.ForeignObject', {
                 display: 'flex',
                 flexDirection: 'column',
                 alignItems: 'center',
-                justifyContent: 'center'
+                justifyContent: 'center',
+                // Prevents the browser from scrolling the page when the user
+                // tries to touch the foreignObject.
+                // Works in Chrome, but not in Safari. If set on the paper,
+                // it works everywhere.
+                // touchAction: 'none'
             },
             children: [{
                 tagName: 'span',
-                textContent: 'First Name'
+                textContent: 'First Name',
```

Comment:
trailing commas are inconsistent in this file (line 11, 60, 64, 70, 84) - do we want to have them, not have them, or does it not matter?

Reply:
We don't enforce it yet, but it should be consistent (at least within the file).